torch/_dynamo/variables/builder.py

```diff
@@ -1724,6 +1728,28 @@ def wrap_tensor(self, value: torch.Tensor):
             self.mark_static_input(value, guard=is_parameter_freezing())
             is_static_input = True
 
+        # Check to install params/bufs which are not input when requested
+        # Assumption: LocalSource is the only way our input is passed
+        # NOTE: These semantics currently will not install nested params/buffers, such as
+        # if a nn.Module is an input to a function, or list of parameters
+        if (
+            config.install_params_as_graph_attr
+            and not (
+                is_from_local_source(source, only_allow_input=True)
+                and not is_source_self(source)
+            )
+            and (
+                is_from_unspecialized_param_buffer_source(source)
```

Comment:
Maybe instead of `is_explicit_member_buffer` this should just be `is_explicit_member`, as that would solve the cases where we have 
```
        class M(torch.nn.Module):
            def __init__(self) -> None:
                super().__init__()
                self.a = torch.ones(3, 3)

            def forward(self):
                return self.a
```

Reply:
In that case, `a` is a member of local source, but the type is not buffer, but tensor so should still be installed (otherwise `test_export.py ExportTests.test_empty` fails)